myceliald/src/constructors/typecast_transformer.rs

```diff
@@ -0,0 +1,52 @@
+use pipe::{config::Map, types::DynSection};
+use section::{command_channel::SectionChannel, message::DataType, SectionError};
+
+pub fn transformer<S: SectionChannel>(
+    config: &Map,
+) -> Result<Box<dyn DynSection<S>>, SectionError> {
+    let column = config
+        .get("column")
+        .ok_or("tagging requires a 'column' name")?
+        .as_str()
+        .ok_or("tagging 'column' name must be a string")?;
+    let target_type = match config
+        .get("target_type")
+        .ok_or("typecasting requires a 'target_type'")?
+        .as_str()
+        .ok_or("tagging 'text' must be set")?
```

Comment:
nit: here the field name says `text` but should be `target_type`

Reply:
Thank you!